llvm/lib/Analysis/AssumptionCache.cpp

```diff
@@ -42,6 +42,12 @@ static cl::opt<bool>
                           cl::desc("Enable verification of assumption cache"),
                           cl::init(false));
 
+// Implemented in ValueTracking.cpp
+void addValueAffectedByCondition(Value *V,
+                                 function_ref<void(Value *)> InsertAffected);
+void findValuesAffectedByCondition(Value *Cond, bool IsAssume,
+                                   function_ref<void(Value *)> InsertAffected);
```

Comment:
Hm, any reason not to export this function in ValueTracking.h?

Reply:
No, initially I had thought we need includes to go the other way.